### DISCUSSION TITLE
run IM on a remote pc

### DISCUSSION THREAD

**Discussion by mmotta19** [ImageMagick/ImageMagick#4519] [Help]
Hello.
I have IM installed on a computer. The installation folder is shared. I run it from other computers and it always worked perfectly.
Recently we need to convert PDF to JPG. I installed Ghostscript and it works locally, but when I run the convert command over the network (\\\\192.168.0.24\imagemagick\convert.exe -density 200 -quality 100 doc.pdf doc.jpg), it gives an error:
convert.exe: RegistryKeyLookuoFailed 'CoderModulesPath' @ error/module.c/GetMagickModulePath/662.
convert.exe: no decode delegate for this image format 'INI' @ error/constitute.c/ReadImage/501.
convert.exe: no images defined 'opc.jpg' @ error/convert.c/ConvertImageCommand/3230.

Is it can be done?

Thanks
Marcelo Motta

**Comment by snibgo** [comment]
That command did not produce those error messages: a message mentions opc.jpg, but that is not in the command.

Exactly what version of IM was installed? I expect this will work if you install the "portable" version, because that doesn't use the registry. As an error mentions the registry, I suspect your version isn't "portable".

**Reply by mmotta19** [reply]
Hello.
I am using ImageMagick version 6.8.8-2 Q8 x86 2014-01-09 http://www.imagemagick.org
I figured it could work that way. I'll take a look at the portable version then...
Thanks!

**Comment by mmotta19** [comment]
Hi.
I ran the test and it worked...

Now I have another problem...
When I run the command "convert.exe -density 200 - quality 100 test.pdf test.jpg", it works but it is generating a large image.
If you put a low density, it compromises the quality...
That way I can convert to jpg and then run the command "convert.exe test.jpg -resize "800x600>" -quality 100 test.jpg" and it works.
However, I tested the direct resize in the command that converts from pdf to jpg, "convert.exe -density 200 - quality 100 -resize "800x600>" test.pdf test.jpg" and it worked, but the image was as flat background in black, different from the original..
Can I do everything in 1 command without compromising the image or will I have to resize the image in another command?

I wait and thank you..
Marcelo Motta
